src/providers/codeforces.cr

```diff
@@ -0,0 +1,37 @@
+class Codeforces < SiteStats
+  @@url_regex = Regex.new("https?://codeforces[.]com/profile/(?<username>\\w+)")
+  # https://codeforces.com/apiHelp
+  @@api_url = "https://codeforces.com/api/user.status?handle=%s&from=1&count=10000"
```

Comment:
Some of the top users had over 7000 submissions already, so I think if we change this 10000 to like 50000 or something it should be safe for the foreseeable future. Other option would be to implement pagination (something similar to what the CodeWars and Exercism providers do), but I think one big request is better than multiple small ones for our purposes.